src/layer/S2L1CLayer.ts

```diff
@@ -12,6 +12,7 @@ export class S2L1CLayer extends AbstractSentinelHubV3WithCCLayer {
 
   protected getWmsGetMapUrlAdditionalParameters(): Record<string, any> {
     return {
+      ...super.getWmsGetMapUrlAdditionalParameters(),
```

Comment:
Actually Landsat8AWSLayer doesn't have this function, so I guess maxCC is not handled correctly when using wms

Reply:
Agreed, nicely spotted! I have now moved `getWmsGetMapUrlAdditionalParameters` to `AbstractSentinelHubV3WithCCLayer` instead of defining on (some) children (`S2L1CLayer`, `S2L2ALayer`, `S3SLSTRLayer`).